lib/ansible/modules/cloud/amazon/ec2_placement_group.py

```diff
@@ -0,0 +1,185 @@
+#!/usr/bin/python
+# Copyright (c) 2017 Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = '''
+---
+module: ec2_placement_group
+short_description: Create or delete an EC2 Placement Group
+description:
+    - Create an EC2 Placement Group; if the placement group already exists,
+      nothing is done. Or, delete an existing placement group. If the placement
+      group is absent, do nothing.
+version_added: "2.5"
+author: "Brad Macpherson (@iiibrad)"
+options:
+  name:
+    description:
+      - The name for the placement group.
+    required: true
+  state:
+    description:
+      - Create or delete placement group.
+    required: false
+    default: present
+    choices: [ 'present', 'absent' ]
+extends_documentation_fragment:
+    - aws
+    - ec2
+'''
+
+EXAMPLES = '''
+# Note: These examples do not set authentication details, see the AWS Guide
+# for details.
+
+# Create a placement group.
+- ec2_placement_group:
+    name: my-cluster
+    state: present
+
+# Delete a placement group.
+- ec2_placement_group:
+    name: my-cluster
+    state: absent
+
+'''
+
+
+RETURN = '''
+placement_group:
+  description: Placement group attributes
+  returned: when state != absent
+  type: complex
+  contains:
+    name:
+      description: PG name
+      type: string
+      sample: my-cluster
+    state:
+      description: PG state
+      type: string
+      sample: "available"
+    strategy:
+      description: PG strategy
+      type: string
+      sample: "cluster"
+
+'''
+
+from ansible.module_utils.aws.core import AnsibleAWSModule
+from ansible.module_utils.ec2 import (boto3_conn,
+                                      ec2_argument_spec,
+                                      get_aws_connection_info,
+                                      HAS_BOTO3)
```

Comment:
HAS_BOTO3 import not required due to use of AnsibleAWSModule

Reply:
Removed